public/pages/workflow_detail/component_details/input_field_list.tsx

```diff
@@ -30,10 +31,10 @@ export function InputFieldList(props: InputFieldListProps) {
               <EuiFlexItem key={idx}>
                 <TextField
                   field={field}
-                  componentId={props.selectedComponent.id}
+                  componentId={props.componentId}
                   onFormChange={props.onFormChange}
                 />
-                <EuiSpacer size="s" />
+                <EuiSpacer size="m" />
```

Comment:
Can we have ENUM constant for the size for `EuiSpacer`?

Reply:
Sure, will add